### ISSUE TITLE
the `.vscode` directory is in the repository

### ISSUE THREAD

**Issue by yegor256** [objectionary/eo-phi-normalizer#560]
I believe, it should be in the `.gitignore`, even through there are other opinions: https://stackoverflow.com/questions/32964920/should-i-commit-the-vscode-folder-to-source-control To me, this directory looks like garbage left in the repo by mistake. Every developer is free to use their own IDE: why only vscode is in the repo?

**Comment by yegor256** [comment]
@deemp please, check

**Comment by deemp** [comment]
@yegor256, no, this file is necessary. I believe that devs should have as similar dev environment as possible when it's preferable.

In our case, `.vscode/settings.json` sets `fourmolu` as the default project formatter. It's preferable that all devs use this formatter.

**Comment by yegor256** [comment]
@deemp what about IntelliJ IDEA developers? What about Vim/Emacs guys? They will suffer without default configurations.

**Comment by deemp** [comment]
@yegor256, we haven't yet received any requests to provide default configurations for devs that use these IDEs.

**Comment by yegor256** [comment]
@deemp this is a matter of taste of every developer -- how to configure their own IDE. Keeping _your personal_ config in the repo looks like something you simply left here by mistake.
